src/Components/test/E2ETestMigration/Tests/BinaryHttpClientTest.cs

```diff
@@ -0,0 +1,92 @@
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Testing;
+using PlaywrightSharp;
+using TestServer;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests
+{
+    public class BinaryHttpClientTest : PlaywrightTestBase,
+        IClassFixture<BasicTestAppServerSiteFixture<CorsStartup>>,
+        IClassFixture<DevHostServerFixture<BasicTestApp.Program>>
+    {
+        private readonly DevHostServerFixture<BasicTestApp.Program> _devHostServerFixture;
+        readonly ServerFixture _apiServerFixture;
+        //IWebElement _appElement;
+        //IWebElement _responseStatus;
+        //IWebElement _responseStatusText;
+        //IWebElement _testOutcome;
+
+        public BinaryHttpClientTest(
+            DevHostServerFixture<BasicTestApp.Program> devHostServerFixture,
+            BasicTestAppServerSiteFixture<CorsStartup> apiServerFixture,
+            ITestOutputHelper output)
+            : base(output)
+        {
+            _devHostServerFixture = devHostServerFixture;
+            _devHostServerFixture.PathBase = "/subdir";
+            _apiServerFixture = apiServerFixture;
+        }
+
+        //protected override void InitializeAsyncCore()
+        //{
+        //    //Browser.Navigate(_devHostServerFixture.RootUri, "/subdir", noReload: true);
+        //    //_appElement = Browser.MountTestComponent<BinaryHttpRequestsComponent>();
+        //}
+
+        [Fact]
+        public async Task CanSendAndReceiveBytes()
+        {
+            if (BrowserManager.IsAvailable(BrowserKind.Chromium))
+            {
+                await using var browser = await BrowserManager.GetBrowserInstance(BrowserKind.Chromium, BrowserContextInfo);
+                var page = await browser.NewPageAsync();
+                await page.GoToAsync(_devHostServerFixture.RootUri + "/subdir/api/data");
+
+/*                var socket = BrowserContextInfo.Pages[page].WebSockets.SingleOrDefault() ??
+                    (await page.WaitForEventAsync(PageEvent.WebSocket)).WebSocket;
+
+                // Receive render batch
+                await socket.WaitForEventAsync(WebSocketEvent.FrameReceived);
+                await socket.WaitForEventAsync(WebSocketEvent.FrameSent);
+
+                // JS interop call to intercept navigation
+                await socket.WaitForEventAsync(WebSocketEvent.FrameReceived);
+                await socket.WaitForEventAsync(WebSocketEvent.FrameSent);
+
+                await page.WaitForSelectorAsync("ul");*/
+
+                await page.CloseAsync();
+            }
+
+
+            //IssueRequest("/subdir/api/data");
+            //Assert.Equal("OK", _responseStatus.Text);
+            //Assert.Equal("OK", _responseStatusText.Text);
+            //Assert.Equal("", _testOutcome.Text);
+        }
+
+        private void IssueRequest()
```

Comment:
I wanted to leave the existing test code there since it might aid in the process of migrating this test to playwright apis later.  All the commended out code is the selenium test code.